api/src/hyp3_api/handlers.py

```diff
@@ -90,13 +90,25 @@ def get_jobs(user, start=None, end=None, status_code=None, name=None):
     return {'jobs': response['Items']}
 
 
+def get_names_for_user(user):
+    table = DYNAMODB_RESOURCE.Table(environ['TABLE_NAME'])
+    key_expression = Key('user_id').eq(user)
+    response = table.query(
+        IndexName='user_id',
+        KeyConditionExpression=key_expression,
+    )
+    names = {record['name'] for record in response['Items'] if 'name' in record}
+    return sorted(list(names))
+
+
 def get_user(user):
     return {
         'user_id': user,
         'quota': {
             'limit': int(environ['MONTHLY_JOB_QUOTA_PER_USER']),
             'remaining': get_remaining_jobs_for_user(user),
         },
+        'names': get_names_for_user(user)
```

Comment:
`user.job_names` is clearer that we're talking about jobs than `user.names`.  That might be a smell that we're coopting `/user` for something that might deserve a separate endpoint, but I'm fine with just clarifying the field name.